README.md

```diff
@@ -15,15 +15,15 @@ We also want to give the right weight to **technical decision-making** and in re
 
 ## Principles
 
-Everytime that we need to choose a framework for a project we will use these principles to guide our choices.
+Everytime that we consider the choice of a framework for a project we will use these principles to guide our choices.
 
```

Comment:
I argue that there is never the "need" of a framework.  There might be the *perception" of the need.

Reply:
You're right! Removing the "need" is part of our purpose after all :D